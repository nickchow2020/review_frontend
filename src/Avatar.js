import { Avatar } from 'antd';
import "./Avatar.css"

const userAvatar = ({url})=>{
  return (
    <>
    <Avatar src={url} className="avatar"/>
    </>
  )
}

export default userAvatar;