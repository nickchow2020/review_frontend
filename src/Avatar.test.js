import React from 'react';
import { render } from '@testing-library/react';
import Avatar from './Avatar';

it("should render", function(){
  render(<Avatar />)
})

it("match snapshot", function(){
  const {asFragment} = render(<Avatar />)

  expect(asFragment()).toMatchSnapshot()
})