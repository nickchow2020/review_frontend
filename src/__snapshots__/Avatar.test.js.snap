// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
<DocumentFragment>
  <span
    class="ant-avatar ant-avatar-circle avatar"
  >
    <span
      class="ant-avatar-string"
      style="transform: scale(1) translateX(-50%);"
    />
  </span>
</DocumentFragment>
`;
